src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/script.js

```diff
@@ -45,50 +45,52 @@ function createElem(doc, tag, path) {
     scriptElement.parentNode.insertBefore(script, scriptElement);
 }
 
-function show(type) {
-    count = 0;
-    for (var key in data) {
-        var row = document.getElementById(key);
-        if ((data[key] &  type) !== 0) {
-            row.style.display = '';
-            row.className = (count++ % 2) ? rowColor : altColor;
-        }
-        else
-            row.style.display = 'none';
+function show(tableId, selected, columns) {
+    if (tableId !== selected) {
+        document.querySelectorAll('div.' + tableId + ':not(.' + selected + ')')
+            .forEach(function(elem) {
+                elem.style.display = 'none';
+            });
     }
-    updateTabs(type);
+    document.querySelectorAll('div.' + selected)
+        .forEach(function(elem, index) {
+            elem.style.display = '';
+            var isEvenRow = index % (columns * 2) < columns;
+            elem.classList.remove(isEvenRow ? rowColor : altColor);
+            elem.classList.add(isEvenRow ? altColor : rowColor);
```

Comment:
I wasn't going to suggest this until I saw the use of these names. Can we fix these weird/bad/non-obvious names for `rowColor` and `altColor`.  Suggest either names like `oddRowColor` `evenRowColor` or `whiteRowColor` `greyRowColor` etc